### ISSUE TITLE
[Bug] Glow-TTS model with out Phoenimizer trained on LJS? 

### ISSUE THREAD

**Issue by EmElleE** [coqui-ai/TTS#513]
**Describe the bug**
Hey so we are just trying to run the default models and test out inference on coqui-ai
We wanted to use glow tts without phonemizer python, since there is GPL-3 license conflicts, with our own work. 

Anyways we have tried loading the models provided and changing the json config to not use phoenmization and it doesn't execute the model at all and complains about the state dictionary.

**To Reproduce**
Steps to reproduce the behaviour:
https://drive.google.com/drive/folders/1gsjRYx4rdSHmyUjgHl1xkP0QRkteC03p download the model from here.
modify the config to say.
"use_phonemes": false

**Expected behavior**
It should load the model and inference can be executed.
Complains about the model dimensions and sizes.

**Additional context**

On a related note: Would it be easy to shim in a lookup dictionary generated by phonemizer for the English language. Is phonemicizing is context-independent? I am ignorant of how that works this is just an idea.

We are training a Glow-TTS model with LJS as we speak, we are just having trouble getting it to converge the loss is still pretty high, Any other tips for this? we will be willing to contribute that model to this project. Thanks!

**Comment by erogol** [comment]
You cannot change a pre-trained model from phonemes to no-phonemes. You need to train a new model.
